content/departments/engineering/dev/enablement/dev-experience/processes.md

```diff
@@ -46,20 +70,21 @@ From the answers, we create a Definition of Done for the item that should includ
 
 ### Tracking
 
-As of 2022-04-28, we use Github projects to keep track of our work.
-
-### Product and Engineering roadmap project
-
-Our quarterly roadmap items are defined in [tracking issues](../../process/tracking_issues.md) on the [Product and Engineering roadmap](https://github.com/orgs/sourcegraph/projects/214/views/14). These tracking issues describe the problem space, requirements, proposed solution, key results, and measures of success; provide links to relevant RFCs, discussions, arifacts, and decisions; and collate the smaller issues that contribute to the execution of our roadmap. In this way, they reflect a high-level view of our progress on a given roadmap item and provide necessary context. These issues have the `team/devx`, `roadmap`, and `tracking` labels.
+As of 2022-04-28, we use GitHub issues and projects to keep track of our work.
 
-### Dev Experience project
+#### Dev Experience project
 
 For a lower level view of our day-to-day progress, we use the [Dev Experience Github project](https://github.com/orgs/sourcegraph/projects/212/views/32). This board automatically imports issues with the `dx` or `team/devx` labels.
 
 ### Google Drive
 
 We collect meeting notes and planning artifacts in a shared [Google Drive folder](https://drive.google.com/drive/folders/1d1scMzzmXM5uCEpKI06U9cc6zPF7g9wE).
 
+#### Scratch documents
+
+We encourage the liberal use of "scratch documents" - temporary, short-lived Google Docs that are used as discussion forums and a dump for ideas.
+Concrete, permanent artefacts should be created from them, e.g. via the [planned work process](#planned-work).
+
```

Comment:
Considering they are located in a public repository, should we mention our scratch notebooks here, too?